openpype/tools/loader/app.py

```diff
@@ -608,14 +608,4 @@ def cli(args):
     # Store settings
     api.Session["AVALON_PROJECT"] = project
 
-    from avalon import pipeline
-
-    # Find the set config
-    _config = pipeline.find_config()
-    if hasattr(_config, "install"):
-        _config.install()
-    else:
-        print("Config `%s` has no function `install`" %
-              _config.__name__)
-
```

Comment:
Does this break the standalone loader? Or does that trigger its own install already? Since I'm not seeing a replacement OpenPype install here?

Reply:
I've moved the `install_openpype_plugins` to the launch logic in tray and I forgot about the `cli` function as we don't use it.